### DISCUSSION TITLE
KEYBOARD PROBLEMS

### DISCUSSION THREAD

**Discussion by RBTCHENTPE** [joncampbell123/dosbox-x#4970] [General]
Would please kindly teach me how to modify the keyboard configuration for following problems?

windows 10 with DosBox-x the keyboard create numbers of the same letter Key-stroked (after released Key one times only) /digit continuously.

**Comment by andy-cymro** [comment]
This happens on Windows 11 for me too. Sometimes single duplications, sometimes the key is virtually held down.
